src/components/ImportForm/SourceSection/SourceSection.tsx

```diff
@@ -94,7 +93,7 @@ const SourceSection: React.FC<SourceSectionProps> = () => {
           const { organization } = gitUrlParse(sourceUrl);
           if (!organization) {
             setValidated(ValidatedOptions.error);
-            setHelpTextInvalid('Not a valid source repository');
+            setHelpTextInvalid("That repository URL isn't quite right. Try again.");
```

Comment:
You also need to update the text in validation utils.

Reply:
added